tests/test_models.py

```diff
@@ -36,6 +35,5 @@ def test_available_for_processing_queryset(self):
         tasks = SNSTask.objects.available_for_processing()
         self.assertEquals(list(tasks), [
             task_ready_past,
-            task_ready,
             task_ready_now,
         ])
```

Comment:
This test became broken when we switched from ordering by auto-id to ordering by `visible_after`. Mysql just happened to continue returning objects in the same order, but postgres is not.